### ISSUE TITLE
No version info included in core-setup repo products in Preview 8

### ISSUE THREAD

**Issue by janvorli** [dotnet/core-setup#7933]
While looking into an issue reported by a customer, I’ve just noticed that we are not adding version information into the native binaries produced by the core-setup repo in Preview 8 anymore (at least for Linux, I haven’t checked others). It was present in Preview 7 and previous versions. 

Preview 8:
```
strings libhostfxr.so | grep "@(#)"
@(#)No version information produced
```
Preview 7:
```
strings libhostfxr.so | grep "@(#)"
@(#)Version 3.0.27912.14 @BuiltBy: root-d34719a418ca @Branch: internal/release/3.0 @SrcCode: https://github.com/dotnet/core-setup/tree/4da6ee6450e4dd43ef8c05925fec22d9ac3892f3
```
This is an important information I use to find out exact commit from which the binary was built when debugging issues.

**Comment by janvorli** [comment]
cc: @jeffschwMSFT

**Comment by dagood** [comment]
This is a duplicate of https://github.com/dotnet/core-setup/issues/7635, fixed for 3.0. (Not in the Preview 9 branch/build.)
